.github/workflows/examples.yml

```diff
@@ -57,7 +57,7 @@ jobs:
       name: Cache plugin dir
       with:
         path: ~/.tflint.d/plugins
-        key: ${{ matrix.os }}-tflint-${{ hashFiles('.tflint.hcl') }}
+        key: ${{ runner.os }}-tflint-${{ hashFiles('.tflint.hcl') }}
```

Comment:
using `runner.os` here since the matrix defined in this workflow only has a parameter for terraform versions